### ISSUE TITLE
Add orientation setting to wheel and engine

### ISSUE THREAD

**Issue by WolfgangFahl** [rc-dukes/dukes#31]
```
# positive - higher values mean go right
wheel.orientation=+

# positive - higher values means going quicker
engine.orientation=+
```

The original orientation settings seem to be negative ...

**Comment by WolfgangFahl** [comment]
Tim and Bert-Jan could you please check whether my assumption is right?

**Comment by WolfgangFahl** [comment]
there are some places where absolute values are used e.g. performBrake - it would be good to get some hints on how to best implement the turning/reversing of orientations of the servos

**Comment by WolfgangFahl** [comment]
I found the following code that seems to try the samething somewhat "automagically":
```
// reverse speed because motor was initially connected the wrong way around
        int diff = Math.abs(engineMap.neutral() - speed);
        if (speed < engineMap.neutral()) {
            speed = engineMap.neutral() + Math.abs(diff);
        } else {
            speed = engineMap.neutral() - Math.abs(diff);
        }
```

**Comment by bertjan** [comment]
If I recall correctly, the speed controller calibrates its zero/idle throttle value to the PWM value that is being sent to it when the speed controller turns on. We experimentally found out the max and min throttle values and calculated speed percentages between idle->max-reverse and idle->max-forward

**Comment by bertjan** [comment]
I also recall that at some point we switched the wiring to the engine. The speed controller we used had a brake function. If you give a large amount of throttle reverse while going forward, it would lock up the wheels to act like a brake. This only worked with wiring in one direction.

**Comment by WolfgangFahl** [comment]
Thx for the hint. I am having issues with the Tamiy TEU-105BK controller that came with the 1:10 car I got from ebay: https://cdn.simba-dickie-group.de/downloads/300057984/300057984_Anleitung_TEU-105BK.pdf Control shows the procedure which does not work as described (or I don't understand how to emulate it with PWM settings yet). I hope to improve the testing options with the current concept of running the ServoCommand as an interface which allows for better debugging, testing and exchanging implementations. I already succesfully implemented the orientation change and it would also be good to have a helper "setup" function for the initialization sequence of ECS. That will be a separate issue. The part you told me for the parts list sold out.  I have now ordered a "Speed Control Runner Plus Reverse Digital 18-36 Turns 80 Amps"  as an alternative. See http://beaverappraisal.com/lrp.pdf
